cypress/integration/Radio.spec.ts

```diff
@@ -0,0 +1,46 @@
+import * as h from '../helpers';
+
+const getRadio = () => {
+  return cy.get(`[type="radio"]`);
+};
+
+describe('Radio', () => {
+  before(() => {
+    h.stories.visit();
+  });
+  ['Default'].forEach(story => {
+    context(`given the '${story}' story is rendered`, () => {
+      beforeEach(() => {
+        h.stories.load('Components|Inputs/Radio/React/Left Label/Radio', story);
```

Comment:
No Radio Group tests?

Reply:
Yep yep, was waiting for your checkbox PR to go in before adding some!